### PR TITLE
Add Lonsonhome QS-Zigbee-C01

### PR DESCRIPTION
**Pull request by miit86**
class QS-Zigbee-C01(CustomDevice):
    """Tuya smart curtain roller shutter."""

    signature = {
        # SizePrefixedSimpleDescriptor(endpoint=1, profile=260, device_type=0x0202, device_version=1, input_clusters=[0, 4, 5, 0x0102], output_clusters=[10, 25]))
        MODELS_INFO: [("_TZ3000_vd43bbfq", "TS130F")],
        ENDPOINTS: {
            1: {
                PROFILE_ID: zha.PROFILE_ID,
                DEVICE_TYPE: zha.DeviceType.WINDOW_COVERING_DEVICE,
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    WindowCovering.cluster_id,
                ],
                OUTPUT_CLUSTERS: [Time.cluster_id, Ota.cluster_id,],
            },
        },
    }
    replacement = {
        ENDPOINTS: {
            1: {
                PROFILE_ID: zha.PROFILE_ID,
                DEVICE_TYPE: zha.DeviceType.WINDOW_COVERING_DEVICE,
                INPUT_CLUSTERS: [
                    Basic.cluster_id,
                    Groups.cluster_id,
                    Scenes.cluster_id,
                    TuyaCoveringCluster,					
                ],
                OUTPUT_CLUSTERS: [Time.cluster_id,Ota.cluster_id],
            },
        },
    }

### CODE REVIEW

File: zhaquirks/tuya/ts130f.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    attributes = {0x8001: ("backlight_mode", t.enum8)}
+
+
+class TuyaCoveringCluster(CustomCluster, WindowCovering):
+    """TuyaSmartCurtainWindowCoveringCluster: Allow to setup Window covering tuya devices."""
+
+    attributes = WindowCovering.attributes.copy()
+    attributes.update({0xF000: ("tuya_moving_state", t.enum8)})
+    attributes.update({0xF001: ("calibration", t.enum8)})
+    attributes.update({0xF002: ("motor_reversal", t.enum8)})
```
Comment (Adminiuga) [inline zhaquirks/tuya/ts130f.py:39]:
```suggestion
    manufacturer_attributes = {
        0xF000: ("tuya_moving_state", t.enum8),
        0xF001: ("calibration", t.enum8),
        0xF002: ("motor_reversal", t.enum8),
    }
```

File: zhaquirks/tuya/ts130f.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+        if attrid == ATTR_CURRENT_POSITION_LIFT_PERCENTAGE:
+            # Invert the percentage value (cf https://github.com/dresden-elektronik/deconz-rest-plugin/issues/3757)
+            value = 100 - value
+        super()._update_attribute(attrid, value)
+
+    async def command(
+        self, command_id, *args, manufacturer=None, expect_reply=True, tsn=None
+    ):
+        """Override default command to invert percent lift value."""
+        if command_id == CMD_GO_TO_LIFT_PERCENTAGE:
```
Comment (Adminiuga) [inline zhaquirks/tuya/ts130f.py:51]: Could you try this instead
```suggestion
    async def go_to_tilt_percentage(
        self, *args, manufacturer=None, expect_reply=True, tsn=None
    ):
        """Override default command to invert percent lift value."""
```

It is going to override just that command.

**Review by Adminiuga — CHANGES_REQUESTED** [review]
Can you please address comments?

**Comment by MattWestb** [comment]
One question is your curtain roller shutter working with the updated quirk (is now using only MODEL and the endpoints cluster for loading so no need for MODELS_INFO for all devices) ?

**Comment by miit86** [comment]
Hi,
Yes it is working with the TS130F
I once started this request, but didn't know the steps/way to proceed.
I just closed the Request.

**Comment by MattWestb** [comment]
Very good and as long tuya is doing all the same tuya way its working in ZHA :-))
And closing the PR then its not needed is good so its no need for it laying around :-)

**Pull request closed without merging**